api/nodes/urls.py

```diff
@@ -1,5 +1,5 @@
-from django.conf.urls import url
 from api.nodes import views
+from django.conf.urls import url
```

Comment:
3rd party imports should go before application imports: https://cosdev.readthedocs.org/en/latest/style_guides/python.html#imports